dex/testing/eth/create-node.sh

```diff
@@ -34,19 +34,25 @@ if [ "${CHAIN_ADDRESS}" != "_" ]; then
   # The mining script may end up mining more or less blocks than specified.
   cat > "${NODES_ROOT}/harness-ctl/mine-${NAME}" <<EOF
 #!/usr/bin/env bash
-  NUM=1
+  NUM=2
   case \$1 in
-      ''|*[!0-9]*)  ;;
+      ''|*[!0-9]*|[0-1])  ;;
       *) NUM=\$1 ;;
   esac
   echo "Mining..."
   BEFORE=\$("${NODES_ROOT}/harness-ctl/${NAME}" attach --exec 'eth.blockNumber')
   "${NODES_ROOT}/harness-ctl/${NAME}" attach --exec 'miner.start()' > /dev/null
-  sleep \$(echo "\$NUM-0.5" | bc)
+  sleep \$(echo "\$NUM-1.8" | bc)
   "${NODES_ROOT}/harness-ctl/${NAME}" attach --exec 'miner.stop()' > /dev/null
+  sleep 1
   AFTER=\$("${NODES_ROOT}/harness-ctl/${NAME}" attach --exec 'eth.blockNumber')
```

Comment:
I don't have one at the moment, but it can happen.  For reference, i had full node notes going here: https://gist.github.com/chappjc/4feefe593f870cd008180c00f72f85ca
I had it running on a cloud machine, in this experiment.  The main takeaway from that is that you need to periodically shutdown and do a `snapshot prune-state` to bring your primary datadir size under control.  Not great for mission critical apps, so hopefully they are working to make that pruning an online/hot process.

Reply:
The other takeaways are that a full node can be prepared overnight on a modest machine with ~400GB disk, split between ssd for indexes and state, and hdd for ancient, but that leaves very little room for chain growth.

Reply:
Idk, taking me longer with "snap"... I think the other method was faster, but I guess it will be faster when it comes to syncing the state at the end?

Reply:
Hmm, that's odd.  The bandwidth use is greatly reduced with snap, so perhaps it's no better for you.